curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-042.md

```diff
@@ -1,34 +1,32 @@
 ---
-id: 6194079d7cbf586eba9539ff
+id: 620175b3710a0951cfa86edf
 title: Step 42
 challengeType: 0
 dashedName: step-42
 ---
 
 # --description--
 
-Your last `.row` element in each section should stand out, as this is your "total" row. The `:last-child` selector allows you to target the matching element that is the final child in a group of siblings.
-
-Create a `.row:last-child` selector and give it a `background-color` property set to `transparent`. Also set the `margin-bottom` property to `30px` to create some extra space between your sections.
+Ensure your years do not get hidden by setting a `z-index` of `999`. Then, give it a `margin` of `0 -2px`, and a `padding` set to `0.5rem calc(1.25rem + 2px) 0.5rem 0`.
 
 # --hints--
 
-You should have a new `.row:last-child` selector.
+Your `#years` selector should have a `z-index` property set to `999`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.row:last-child'));
+assert(new __helpers.CSSHelp(document).getStyle('#years')?.getPropertyValue('z-index') === '999');
 ```
 
-Your `.row:last-child` selector should have a `background-color` property set to `transparent`.
+Your `#years` selector should have a `margin` property set to `0 -2px`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.row:last-child')?.backgroundColor === 'transparent');
+assert(new __helpers.CSSHelp(document).getStyle('#years')?.getPropertyValue('margin') === '0px -2px');
 ```
 
-Your `.row:last-child` selector should have a `margin-bottom` property set to `30px`.
+Your `#years` selector should have a `padding` property set to `0.5rem calc(1.25rem + 2px) 0.5rem 0`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.row:last-child')?.marginBottom === '30px');
+assert(new __helpers.CSSHelp(document).getStyle('#years')?.getPropertyValue('padding') === '0.5rem calc(1.25rem + 2px) 0.5rem 0px');
```

Comment:
Get this, Firefox returns this property as `0.5rem calc(2px + 1.25rem) 0.5rem 0px`. :eyes: